spec/std/http/cookie_spec.cr

```diff
@@ -35,16 +35,27 @@ module HTTP
         expect_raises IO::Error, "Invalid cookie name" do
           HTTP::Cookie.new("", "")
         end
+
```

Comment:
What's the reason for these unrelated whitespace changes?

Reply:
No real reason other than it looked bad.

Ref: https://github.com/crystal-lang/crystal/issues/9804.

Reply:
Then it probably shouldn't be part of this PR.